packages/package_info_plus/package_info_plus_windows/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: package_info_plus_windows
 description: Windows implementation of the package_info_plus plugin
-version: 1.0.5
+version: 1.0.6
```

Comment:
It looks like `ffi` requires Dart 2.17 as minimum supported version since `2.0.0` release. Thus, w ned to update this pubspec's environment section accordingly and because of this bump we would have to bump `package_info_plus_windows` to `2.0.0` instead of `1.0.6`.

Reply:
I changed version `2.0.0` instead of `1.0.6`.